Description: Upstream changes introduced in version 7.3-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 org-mode (7.3-1) unstable; urgency=low
 .
   * New upstream release (Closes: #602196).
 .
 The person named in the Author field signed this changelog entry.
Author: Sebastien Delafond <seb@debian.org>
Bug-Debian: http://bugs.debian.org/602196

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- org-mode-7.3.orig/doc/org.vrs
+++ org-mode-7.3/doc/org.vrs
@@ -1,255 +1,261 @@
 \initial {C}
-\entry {\code {cdlatex-simplify-sub-super-scripts}}{114}
-\entry {\code {constants-unit-system}}{22, 163}
+\entry {\code {cdlatex-simplify-sub-super-scripts}}{115}
+\entry {\code {constants-unit-system}}{23, 164}
 \initial {H}
-\entry {\code {htmlize-output-type}}{102}
+\entry {\code {htmlize-output-type}}{103}
 \initial {L}
-\entry {\code {La\TeX {}-verbatim-environments}}{174}
+\entry {\code {La\TeX {}-verbatim-environments}}{176}
 \initial {O}
-\entry {\code {org-adapt-indentation}}{165}
-\entry {\code {org-agenda-add-entry-text-maxlines}}{102}
-\entry {\code {org-agenda-columns-add-appointments-to-effort-sum}}{72}
-\entry {\code {org-agenda-confirm-kill}}{97}
-\entry {\code {org-agenda-custom-commands}}{9, 100, 101, 102, 179}
-\entry {\code {org-agenda-diary-file}}{99}
-\entry {\code {org-agenda-dim-blocked-tasks}}{41}
-\entry {\code {org-agenda-entry-text-maxlines}}{94}
-\entry {\code {org-agenda-exporter-settings}}{99, 102}
-\entry {\code {org-agenda-files}}{83, 92, 131}
-\entry {\code {org-agenda-filter-preset}}{95}
-\entry {\code {org-agenda-log-mode-items}}{94}
-\entry {\code {org-agenda-ndays}}{85, 94}
-\entry {\code {org-agenda-overriding-header}}{178}
-\entry {\code {org-agenda-prefix-format}}{91}
-\entry {\code {org-agenda-restore-windows-after-quit}}{83}
-\entry {\code {org-agenda-show-inherited-tags}}{97}
-\entry {\code {org-agenda-skip-archived-trees}}{82}
-\entry {\code {org-agenda-skip-function}}{178, 182}
-\entry {\code {org-agenda-skip-scheduled-if-done}}{65}
-\entry {\code {org-agenda-sorting-strategy}}{92}
-\entry {\code {org-agenda-start-with-clockreport-mode}}{94}
-\entry {\code {org-agenda-start-with-entry-text-mode}}{94}
-\entry {\code {org-agenda-start-with-follow-mode}}{93}
-\entry {\code {org-agenda-tags-todo-honor-ignore-options}}{88}
-\entry {\code {org-agenda-text-search-extra-files}}{85, 90}
-\entry {\code {org-agenda-time-grid}}{92, 94}
-\entry {\code {org-agenda-todo-ignore-deadlines}}{87}
-\entry {\code {org-agenda-todo-ignore-scheduled}}{87}
-\entry {\code {org-agenda-todo-ignore-with-date}}{87}
-\entry {\code {org-agenda-todo-list-sublevels}}{45, 87}
-\entry {\code {org-agenda-use-time-grid}}{92, 94}
-\entry {\code {org-agenda-window-setup}}{83}
-\entry {\code {org-archive-default-command}}{81, 97}
-\entry {\code {org-archive-location}}{81, 161}
-\entry {\code {org-archive-save-context-info}}{81}
-\entry {\code {org-attach-directory}}{78}
-\entry {\code {org-attach-method}}{78}
-\entry {\code {org-babel-default-header-args}}{144, 145}
-\entry {\code {org-calc-default-modes}}{23}
-\entry {\code {org-clock-idle-time}}{70}
-\entry {\code {org-clock-into-drawer}}{67}
-\entry {\code {org-clock-modeline-total}}{68}
-\entry {\code {org-coderef-label-format}}{109}
-\entry {\code {org-columns-default-format}}{58, 72, 95, 104}
-\entry {\code {org-columns-skip-archived-trees}}{82}
-\entry {\code {org-combined-agenda-icalendar-file}}{132}
-\entry {\code {org-confirm-babel-evaluate}}{160}
-\entry {\code {org-confirm-elisp-link-function}}{160}
-\entry {\code {org-confirm-shell-link-function}}{160}
-\entry {\code {org-create-file-search-functions}}{36}
-\entry {\code {org-ctrl-c-ctrl-c-hook}}{172}
+\entry {\code {org-adapt-indentation}}{167}
+\entry {\code {org-agenda-add-entry-text-maxlines}}{103}
+\entry {\code {org-agenda-columns-add-appointments-to-effort-sum}}{73}
+\entry {\code {org-agenda-confirm-kill}}{98}
+\entry {\code {org-agenda-custom-commands}}{10, 101, 102, 103, 181}
+\entry {\code {org-agenda-diary-file}}{100}
+\entry {\code {org-agenda-dim-blocked-tasks}}{42}
+\entry {\code {org-agenda-entry-text-maxlines}}{95}
+\entry {\code {org-agenda-exporter-settings}}{100, 103}
+\entry {\code {org-agenda-files}}{84, 93, 133}
+\entry {\code {org-agenda-filter-preset}}{96}
+\entry {\code {org-agenda-log-mode-items}}{95}
+\entry {\code {org-agenda-ndays}}{86, 95}
+\entry {\code {org-agenda-overriding-header}}{180}
+\entry {\code {org-agenda-prefix-format}}{92}
+\entry {\code {org-agenda-restore-windows-after-quit}}{84}
+\entry {\code {org-agenda-show-inherited-tags}}{98}
+\entry {\code {org-agenda-skip-archived-trees}}{83}
+\entry {\code {org-agenda-skip-function}}{180, 184}
+\entry {\code {org-agenda-skip-scheduled-if-done}}{66}
+\entry {\code {org-agenda-sorting-strategy}}{93}
+\entry {\code {org-agenda-start-with-clockreport-mode}}{95}
+\entry {\code {org-agenda-start-with-entry-text-mode}}{95}
+\entry {\code {org-agenda-start-with-follow-mode}}{94}
+\entry {\code {org-agenda-tags-todo-honor-ignore-options}}{89}
+\entry {\code {org-agenda-text-search-extra-files}}{86, 91}
+\entry {\code {org-agenda-time-grid}}{93, 96}
+\entry {\code {org-agenda-todo-ignore-deadlines}}{88}
+\entry {\code {org-agenda-todo-ignore-scheduled}}{88}
+\entry {\code {org-agenda-todo-ignore-with-date}}{88}
+\entry {\code {org-agenda-todo-list-sublevels}}{47, 88}
+\entry {\code {org-agenda-use-time-grid}}{93, 96}
+\entry {\code {org-agenda-window-setup}}{84}
+\entry {\code {org-archive-default-command}}{82, 98}
+\entry {\code {org-archive-location}}{82, 162}
+\entry {\code {org-archive-save-context-info}}{82}
+\entry {\code {org-attach-directory}}{79}
+\entry {\code {org-attach-method}}{80}
+\entry {\code {org-babel-default-header-args}}{145, 146}
+\entry {\code {org-calc-default-modes}}{24}
+\entry {\code {org-clock-idle-time}}{72}
+\entry {\code {org-clock-into-drawer}}{69}
+\entry {\code {org-clock-modeline-total}}{69}
+\entry {\code {org-coderef-label-format}}{110}
+\entry {\code {org-columns-default-format}}{59, 73, 96, 105}
+\entry {\code {org-columns-skip-archived-trees}}{83}
+\entry {\code {org-combined-agenda-icalendar-file}}{133}
+\entry {\code {org-confirm-babel-evaluate}}{162}
+\entry {\code {org-confirm-elisp-link-function}}{162}
+\entry {\code {org-confirm-shell-link-function}}{162}
+\entry {\code {org-create-file-search-functions}}{37}
+\entry {\code {org-ctrl-c-ctrl-c-hook}}{174}
 \entry {\code {org-ctrl-k-protect-subtree}}{5}
 \entry {\code {org-cycle-emulate-tab}}{5}
 \entry {\code {org-cycle-global-at-bob}}{5}
-\entry {\code {org-cycle-include-plain-lists}}{11}
-\entry {\code {org-cycle-open-archived-trees}}{81}
+\entry {\code {org-cycle-include-plain-lists}}{12}
+\entry {\code {org-cycle-open-archived-trees}}{83}
 \entry {\code {org-cycle-separator-lines}}{5}
-\entry {\code {org-deadline-warning-days}}{64, 66}
-\entry {\code {org-default-notes-file}}{74, 76}
-\entry {\code {org-default-priority}}{45, 161}
-\entry {\code {org-display-custom-times}}{64, 135}
-\entry {\code {org-display-internal-link-with-indirect-buffer}}{34}
-\entry {\code {org-disputed-keys}}{168}
-\entry {\code {org-done \r {(face)}}}{40}
-\entry {\code {org-drawers}}{12, 161}
-\entry {\code {org-effort-property}}{71}
-\entry {\code {org-empty-line-terminates-plain-lists}}{10}
-\entry {\code {org-enable-table-editor}}{16}
-\entry {\code {org-enforce-todo-dependencies}}{41}
-\entry {\code {org-entities}}{111, 128}
-\entry {\code {org-execute-file-search-functions}}{36}
-\entry {\code {org-export-ascii-links-to-notes}}{118}
-\entry {\code {org-export-author-info}}{135}
-\entry {\code {org-export-creator-info}}{135}
-\entry {\code {org-export-default-language}}{115, 135}
-\entry {\code {org-export-docbook-default-image-attributes}}{128}
-\entry {\code {org-export-docbook-doctype}}{128}
-\entry {\code {org-export-docbook-inline-image-extensions}}{128}
-\entry {\code {org-export-docbook-xsl-fo-proc-command}}{127}
-\entry {\code {org-export-docbook-xslt-proc-command}}{127}
-\entry {\code {org-export-docbook-xslt-stylesheet}}{127}
-\entry {\code {org-export-email}}{135}
-\entry {\code {org-export-exclude-tags}}{115, 135}
-\entry {\code {org-export-headline-levels}}{106, 135}
-\entry {\code {org-export-highlight-first-table-line}}{135}
-\entry {\code {org-export-html-auto-postamble}}{135}
-\entry {\code {org-export-html-auto-preamble}}{135}
-\entry {\code {org-export-html-expand}}{135}
-\entry {\code {org-export-html-extension}}{135}
-\entry {\code {org-export-html-extra}}{121}
-\entry {\code {org-export-html-inline-images}}{119, 135}
-\entry {\code {org-export-html-link-home}}{135}
-\entry {\code {org-export-html-link-org-files-as-html}}{135}
-\entry {\code {org-export-html-link-up}}{135}
-\entry {\code {org-export-html-postamble}}{135}
-\entry {\code {org-export-html-preamble}}{135}
-\entry {\code {org-export-html-style}}{121, 135}
-\entry {\code {org-export-html-style-default}}{121}
-\entry {\code {org-export-html-style-extra}}{135}
-\entry {\code {org-export-html-style-include-default}}{121, 135}
-\entry {\code {org-export-html-table-tag}}{119, 135}
-\entry {\code {org-export-html-tag-class-prefix}}{120}
-\entry {\code {org-export-html-todo-kwd-class-prefix}}{120}
-\entry {\code {org-export-html-use-infojs}}{122}
-\entry {\code {org-export-html-with-timestamp}}{135}
-\entry {\code {org-export-latex-classes}}{123}
-\entry {\code {org-export-latex-default-class}}{123}
-\entry {\code {org-export-latex-default-packages-alist}}{123}
-\entry {\code {org-export-latex-packages-alist}}{123}
-\entry {\code {org-export-preserve-breaks}}{135}
-\entry {\code {org-export-publishing-directory}}{135}
-\entry {\code {org-export-run-in-background}}{117}
-\entry {\code {org-export-section-number-format}}{135}
-\entry {\code {org-export-select-tags}}{115, 135}
-\entry {\code {org-export-skip-text-before-1st-heading}}{106, 135}
-\entry {\code {org-export-taskjuggler-default-reports}}{130}
-\entry {\code {org-export-taskjuggler-project-tag}}{129}
-\entry {\code {org-export-taskjuggler-resource-tag}}{129}
-\entry {\code {org-export-with-archived-trees}}{82, 135}
-\entry {\code {org-export-with-drawers}}{135}
-\entry {\code {org-export-with-emphasize}}{135}
-\entry {\code {org-export-with-fixed-width}}{135}
-\entry {\code {org-export-with-footnotes}}{135}
-\entry {\code {org-export-with-LaTeX-fragments}}{135}
-\entry {\code {org-export-with-priority}}{135}
-\entry {\code {org-export-with-section-numbers}}{135}
-\entry {\code {org-export-with-special-strings}}{135}
-\entry {\code {org-export-with-sub-superscripts}}{112, 135}
-\entry {\code {org-export-with-tables}}{135}
-\entry {\code {org-export-with-tags}}{135}
-\entry {\code {org-export-with-TeX-macros}}{135}
-\entry {\code {org-export-with-timestamps}}{135}
-\entry {\code {org-export-with-toc}}{106, 135}
-\entry {\code {org-export-with-todo-keywords}}{135}
-\entry {\code {org-fast-tag-selection-include-todo}}{40}
-\entry {\code {org-fast-tag-selection-single-key}}{51}
-\entry {\code {org-file-apps}}{34, 79}
-\entry {\code {org-footnote-auto-adjust}}{14, 163}
-\entry {\code {org-footnote-auto-label}}{14, 163}
-\entry {\code {org-footnote-define-inline}}{14, 163}
-\entry {\code {org-footnote-section}}{14}
-\entry {\code {org-format-latex-header}}{112}
-\entry {\code {org-format-latex-options}}{113}
-\entry {\code {org-from-is-user-regexp}}{78}
-\entry {\code {org-global-properties}}{53, 72}
+\entry {\code {org-deadline-warning-days}}{66, 67}
+\entry {\code {org-default-notes-file}}{75, 77}
+\entry {\code {org-default-priority}}{46, 163}
+\entry {\code {org-display-custom-times}}{65, 136}
+\entry {\code {org-display-internal-link-with-indirect-buffer}}{35}
+\entry {\code {org-disputed-keys}}{170}
+\entry {\code {org-done \r {(face)}}}{41}
+\entry {\code {org-drawers}}{13, 163}
+\entry {\code {org-effort-property}}{73}
+\entry {\code {org-empty-line-terminates-plain-lists}}{11}
+\entry {\code {org-enable-table-editor}}{17}
+\entry {\code {org-enforce-todo-dependencies}}{42}
+\entry {\code {org-entities}}{112, 130}
+\entry {\code {org-execute-file-search-functions}}{37}
+\entry {\code {org-export-ascii-links-to-notes}}{119}
+\entry {\code {org-export-author-info}}{136}
+\entry {\code {org-export-creator-info}}{136}
+\entry {\code {org-export-default-language}}{116, 136}
+\entry {\code {org-export-docbook-default-image-attributes}}{129}
+\entry {\code {org-export-docbook-doctype}}{130}
+\entry {\code {org-export-docbook-inline-image-extensions}}{130}
+\entry {\code {org-export-docbook-xsl-fo-proc-command}}{128}
+\entry {\code {org-export-docbook-xslt-proc-command}}{128}
+\entry {\code {org-export-docbook-xslt-stylesheet}}{128}
+\entry {\code {org-export-email}}{136}
+\entry {\code {org-export-exclude-tags}}{116, 136}
+\entry {\code {org-export-headline-levels}}{107, 136}
+\entry {\code {org-export-highlight-first-table-line}}{136}
+\entry {\code {org-export-html-auto-postamble}}{136}
+\entry {\code {org-export-html-auto-preamble}}{136}
+\entry {\code {org-export-html-expand}}{136}
+\entry {\code {org-export-html-extension}}{136}
+\entry {\code {org-export-html-extra}}{122}
+\entry {\code {org-export-html-inline-images}}{120, 136}
+\entry {\code {org-export-html-link-home}}{136}
+\entry {\code {org-export-html-link-org-files-as-html}}{136}
+\entry {\code {org-export-html-link-up}}{136}
+\entry {\code {org-export-html-postamble}}{136}
+\entry {\code {org-export-html-preamble}}{136}
+\entry {\code {org-export-html-style}}{122, 136}
+\entry {\code {org-export-html-style-default}}{122}
+\entry {\code {org-export-html-style-extra}}{136}
+\entry {\code {org-export-html-style-include-default}}{122, 136}
+\entry {\code {org-export-html-table-tag}}{120, 136}
+\entry {\code {org-export-html-tag-class-prefix}}{122}
+\entry {\code {org-export-html-todo-kwd-class-prefix}}{122}
+\entry {\code {org-export-html-use-infojs}}{124}
+\entry {\code {org-export-html-with-timestamp}}{136}
+\entry {\code {org-export-latex-classes}}{125}
+\entry {\code {org-export-latex-default-class}}{125}
+\entry {\code {org-export-latex-default-packages-alist}}{125}
+\entry {\code {org-export-latex-packages-alist}}{125}
+\entry {\code {org-export-preserve-breaks}}{136}
+\entry {\code {org-export-publishing-directory}}{136}
+\entry {\code {org-export-run-in-background}}{118}
+\entry {\code {org-export-section-number-format}}{136}
+\entry {\code {org-export-select-tags}}{116, 136}
+\entry {\code {org-export-skip-text-before-1st-heading}}{107, 136}
+\entry {\code {org-export-taskjuggler-default-reports}}{132}
+\entry {\code {org-export-taskjuggler-project-tag}}{131}
+\entry {\code {org-export-taskjuggler-resource-tag}}{131}
+\entry {\code {org-export-with-archived-trees}}{83, 136}
+\entry {\code {org-export-with-drawers}}{136}
+\entry {\code {org-export-with-emphasize}}{136}
+\entry {\code {org-export-with-fixed-width}}{136}
+\entry {\code {org-export-with-footnotes}}{136}
+\entry {\code {org-export-with-LaTeX-fragments}}{114, 136}
+\entry {\code {org-export-with-priority}}{136}
+\entry {\code {org-export-with-section-numbers}}{136}
+\entry {\code {org-export-with-special-strings}}{136}
+\entry {\code {org-export-with-sub-superscripts}}{113, 136}
+\entry {\code {org-export-with-tables}}{136}
+\entry {\code {org-export-with-tags}}{136}
+\entry {\code {org-export-with-TeX-macros}}{136}
+\entry {\code {org-export-with-timestamps}}{136}
+\entry {\code {org-export-with-toc}}{107, 136}
+\entry {\code {org-export-with-todo-keywords}}{136}
+\entry {\code {org-fast-tag-selection-include-todo}}{41}
+\entry {\code {org-fast-tag-selection-single-key}}{52}
+\entry {\code {org-file-apps}}{35, 80}
+\entry {\code {org-footnote-auto-adjust}}{15, 165}
+\entry {\code {org-footnote-auto-label}}{15, 165}
+\entry {\code {org-footnote-define-inline}}{15, 165}
+\entry {\code {org-footnote-section}}{15}
+\entry {\code {org-format-latex-header}}{113}
+\entry {\code {org-format-latex-options}}{114}
+\entry {\code {org-from-is-user-regexp}}{79}
+\entry {\code {org-global-properties}}{54, 73}
 \entry {\code {org-goto-auto-isearch}}{7}
 \entry {\code {org-goto-interface}}{7}
-\entry {\code {org-hide \r {(face)}}}{166}
-\entry {\code {org-hide-block-startup}}{13}
-\entry {\code {org-hide-leading-stars}}{163, 165}
-\entry {\code {org-hierarchical-checkbox-statistics}}{47}
-\entry {\code {org-hierarchical-todo-statistics}}{46}
-\entry {\code {org-highest-priority}}{45, 161}
-\entry {\code {org-icalendar-categories}}{131}
-\entry {\code {org-icalendar-include-body}}{132}
-\entry {\code {org-icalendar-include-todo}}{131}
-\entry {\code {org-icalendar-store-UID}}{131}
-\entry {\code {org-icalendar-use-deadline}}{131}
-\entry {\code {org-icalendar-use-scheduled}}{131}
-\entry {\code {org-imenu-depth}}{167}
-\entry {\code {org-infojs-options}}{122}
+\entry {\code {org-hide \r {(face)}}}{167}
+\entry {\code {org-hide-block-startup}}{14}
+\entry {\code {org-hide-leading-stars}}{164, 167}
+\entry {\code {org-hierarchical-checkbox-statistics}}{48}
+\entry {\code {org-hierarchical-todo-statistics}}{47}
+\entry {\code {org-highest-priority}}{46, 163}
+\entry {\code {org-icalendar-alarm-time}}{133}
+\entry {\code {org-icalendar-categories}}{133}
+\entry {\code {org-icalendar-include-body}}{133}
+\entry {\code {org-icalendar-include-todo}}{133}
+\entry {\code {org-icalendar-store-UID}}{133}
+\entry {\code {org-icalendar-use-deadline}}{133}
+\entry {\code {org-icalendar-use-scheduled}}{133}
+\entry {\code {org-imenu-depth}}{169}
+\entry {\code {org-infojs-options}}{124}
 \entry {\code {org-insert-mode-line-in-empty-file}}{3}
 \entry {\code {org-irc-link-to-logs}}{33}
-\entry {\code {org-keep-stored-link-after-insertion}}{33}
-\entry {\code {org-latex-low-levels}}{123}
-\entry {\code {org-link-abbrev-alist}}{35, 161}
-\entry {\code {org-link-to-org-use-id}}{32}
+\entry {\code {org-keep-stored-link-after-insertion}}{34}
+\entry {\code {org-latex-low-levels}}{124}
+\entry {\code {org-link-abbrev-alist}}{36, 163}
+\entry {\code {org-link-to-org-use-id}}{33}
+\entry {\code {org-list-automatic-rules}}{12, 13, 47}
 \entry {\code {org-list-demote-modify-bullet}}{11}
-\entry {\code {org-log-done}}{43, 94, 162}
-\entry {\code {org-log-into-drawer}}{42, 97}
-\entry {\code {org-log-note-clock-out}}{68, 162}
-\entry {\code {org-log-refile}}{80}
-\entry {\code {org-log-repeat}}{66, 162}
-\entry {\code {org-log-states-order-reversed}}{42}
-\entry {\code {org-lowest-priority}}{45, 161}
-\entry {\code {org-M-RET-may-split-line}}{7, 11}
-\entry {\code {org-odd-levels-only}}{88, 163, 166, 178}
-\entry {\code {org-outline-path-complete-in-steps}}{80}
-\entry {\code {org-overriding-columns-format}}{104}
-\entry {\code {org-popup-calendar-for-date-prompt}}{63}
-\entry {\code {org-priority-faces}}{45}
-\entry {\code {org-priority-start-cycle-with-default}}{45}
-\entry {\code {org-property-allowed-value-functions}}{181}
-\entry {\code {org-publish-project-alist}}{133, 136}
-\entry {\code {org-publish-use-timestamps-flag}}{139}
-\entry {\code {org-put-time-stamp-overlays}}{163}
-\entry {\code {org-read-date-display-live}}{64}
-\entry {\code {org-read-date-prefer-future}}{62}
-\entry {\code {org-refile-allow-creating-parent-nodes}}{80}
-\entry {\code {org-refile-targets}}{80}
-\entry {\code {org-refile-use-cache}}{80}
-\entry {\code {org-refile-use-outline-path}}{80}
-\entry {\code {org-remove-highlights-with-change}}{9, 68}
-\entry {\code {org-replace-disputed-keys}}{168}
-\entry {\code {org-reverse-note-order}}{80}
-\entry {\code {org-show-entry-below}}{9}
-\entry {\code {org-show-following-heading}}{9}
-\entry {\code {org-show-hierarchy-above}}{9}
-\entry {\code {org-show-siblings}}{9}
-\entry {\code {org-sort-agenda-noeffort-is-high}}{95}
-\entry {\code {org-sparse-tree-open-archived-trees}}{81}
+\entry {\code {org-list-end-regexp}}{11}
+\entry {\code {org-list-ending-method}}{11}
+\entry {\code {org-log-done}}{44, 95, 164}
+\entry {\code {org-log-into-drawer}}{43, 98}
+\entry {\code {org-log-note-clock-out}}{69, 164}
+\entry {\code {org-log-refile}}{81}
+\entry {\code {org-log-repeat}}{68, 164}
+\entry {\code {org-log-states-order-reversed}}{43}
+\entry {\code {org-lowest-priority}}{46, 163}
+\entry {\code {org-M-RET-may-split-line}}{7, 12}
+\entry {\code {org-odd-levels-only}}{89, 164, 167, 180}
+\entry {\code {org-outline-path-complete-in-steps}}{81}
+\entry {\code {org-overriding-columns-format}}{105}
+\entry {\code {org-plain-list-ordered-item-terminator}}{11, 13}
+\entry {\code {org-popup-calendar-for-date-prompt}}{65}
+\entry {\code {org-priority-faces}}{46}
+\entry {\code {org-priority-start-cycle-with-default}}{46}
+\entry {\code {org-property-allowed-value-functions}}{183}
+\entry {\code {org-publish-project-alist}}{134, 137}
+\entry {\code {org-publish-use-timestamps-flag}}{140}
+\entry {\code {org-put-time-stamp-overlays}}{164}
+\entry {\code {org-read-date-display-live}}{65}
+\entry {\code {org-read-date-prefer-future}}{64}
+\entry {\code {org-refile-allow-creating-parent-nodes}}{81}
+\entry {\code {org-refile-targets}}{81}
+\entry {\code {org-refile-use-cache}}{81}
+\entry {\code {org-refile-use-outline-path}}{81}
+\entry {\code {org-remove-highlights-with-change}}{10, 70}
+\entry {\code {org-replace-disputed-keys}}{170}
+\entry {\code {org-reverse-note-order}}{81}
+\entry {\code {org-show-entry-below}}{10}
+\entry {\code {org-show-following-heading}}{10}
+\entry {\code {org-show-hierarchy-above}}{10}
+\entry {\code {org-show-siblings}}{10}
+\entry {\code {org-sort-agenda-noeffort-is-high}}{96}
+\entry {\code {org-sparse-tree-open-archived-trees}}{83}
 \entry {\code {org-special-ctrl-a/e}}{5}
 \entry {\code {org-special-ctrl-k}}{5}
-\entry {\code {org-speed-commands-user}}{159}
-\entry {\code {org-startup-align-all-tables}}{19, 162}
-\entry {\code {org-startup-folded}}{6, 162}
-\entry {\code {org-startup-indented}}{162}
-\entry {\code {org-store-link-functions}}{171}
-\entry {\code {org-stuck-projects}}{90}
-\entry {\code {org-support-shift-select}}{12, 168}
-\entry {\code {org-table-auto-blank-field}}{16}
-\entry {\code {org-table-copy-increment}}{18}
-\entry {\code {org-table-export-default-format}}{18}
-\entry {\code {org-table-formula}}{161}
-\entry {\code {org-table-formula-constants}}{22, 161, 167}
-\entry {\code {org-table-use-standard-references}}{25}
-\entry {\code {org-tag-alist}}{50, 163}
-\entry {\code {org-tag-faces}}{49}
-\entry {\code {org-tag-persistent-alist}}{50}
-\entry {\code {org-tags-column}}{49}
-\entry {\code {org-tags-exclude-from-inheritance}}{49}
-\entry {\code {org-tags-match-list-sublevels}}{49, 52, 55, 88}
-\entry {\code {org-time-stamp-custom-formats}}{64}
-\entry {\code {org-time-stamp-overlay-formats}}{163}
-\entry {\code {org-time-stamp-rounding-minutes}}{62}
-\entry {\code {org-todo \r {(face)}}}{40}
-\entry {\code {org-todo-keyword-faces}}{40}
-\entry {\code {org-todo-keywords}}{37, 38, 87, 164}
-\entry {\code {org-todo-repeat-to-state}}{66}
-\entry {\code {org-todo-state-tags-triggers}}{38}
-\entry {\code {org-track-ordered-property-with-tag}}{41, 47}
-\entry {\code {org-treat-insert-todo-heading-as-state-change}}{7}
-\entry {\code {org-treat-S-cursor-todo-selection-as-state-change}}{37}
-\entry {\code {org-use-property-inheritance}}{55, 132, 181}
-\entry {\code {org-use-speed-commands}}{159}
-\entry {\code {org-use-tag-inheritance}}{49}
+\entry {\code {org-speed-commands-user}}{161}
+\entry {\code {org-startup-align-all-tables}}{20, 164}
+\entry {\code {org-startup-folded}}{6, 163}
+\entry {\code {org-startup-indented}}{163}
+\entry {\code {org-startup-with-inline-images}}{35, 164}
+\entry {\code {org-store-link-functions}}{173}
+\entry {\code {org-stuck-projects}}{91}
+\entry {\code {org-support-shift-select}}{12, 13, 170}
+\entry {\code {org-table-auto-blank-field}}{17}
+\entry {\code {org-table-copy-increment}}{19}
+\entry {\code {org-table-export-default-format}}{19}
+\entry {\code {org-table-formula}}{163}
+\entry {\code {org-table-formula-constants}}{23, 163, 169}
+\entry {\code {org-table-use-standard-references}}{26}
+\entry {\code {org-tag-alist}}{51, 165}
+\entry {\code {org-tag-faces}}{50}
+\entry {\code {org-tag-persistent-alist}}{51}
+\entry {\code {org-tags-column}}{50}
+\entry {\code {org-tags-exclude-from-inheritance}}{50}
+\entry {\code {org-tags-match-list-sublevels}}{50, 53, 56, 89}
+\entry {\code {org-time-stamp-custom-formats}}{65}
+\entry {\code {org-time-stamp-overlay-formats}}{164}
+\entry {\code {org-time-stamp-rounding-minutes}}{63}
+\entry {\code {org-todo \r {(face)}}}{41}
+\entry {\code {org-todo-keyword-faces}}{41}
+\entry {\code {org-todo-keywords}}{38, 39, 88, 165}
+\entry {\code {org-todo-repeat-to-state}}{67}
+\entry {\code {org-todo-state-tags-triggers}}{39}
+\entry {\code {org-track-ordered-property-with-tag}}{42, 49}
+\entry {\code {org-treat-insert-todo-heading-as-state-change}}{8}
+\entry {\code {org-treat-S-cursor-todo-selection-as-state-change}}{38}
+\entry {\code {org-use-property-inheritance}}{56, 133, 183}
+\entry {\code {org-use-speed-commands}}{161}
+\entry {\code {org-use-tag-inheritance}}{50}
 \entry {\code {org-yank-adjusted-subtrees}}{8}
 \entry {\code {org-yank-folded-subtrees}}{8}
 \initial {P}
-\entry {\code {parse-time-months}}{63}
-\entry {\code {parse-time-weekdays}}{63}
-\entry {\code {ps-landscape-mode}}{102}
-\entry {\code {ps-number-of-columns}}{102}
+\entry {\code {parse-time-months}}{64}
+\entry {\code {parse-time-weekdays}}{64}
+\entry {\code {ps-landscape-mode}}{103}
+\entry {\code {ps-number-of-columns}}{103}
 \initial {U}
-\entry {\code {user-full-name}}{115, 135}
-\entry {\code {user-mail-address}}{115, 135}
+\entry {\code {user-full-name}}{116, 136}
+\entry {\code {user-mail-address}}{116, 136}
--- org-mode-7.3.orig/doc/org.pgs
+++ org-mode-7.3/doc/org.pgs
@@ -1,4 +1,4 @@
 \initial {C}
-\entry {\code {constants.el}}{22}
+\entry {\code {constants.el}}{23}
 \initial {G}
 \entry {\code {GTD, Getting Things Done}}{1}
